customer-samples/AspNetCoreWithSerilog/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.css

```diff
@@ -0,0 +1,3719 @@
+/*!
```

Comment:
I would like it if we remove all the extra static files and keep the js/css super simple. Alternatively, we should consider move "customer samples" into another repo altogether.